test/example_data.dart

```diff
@@ -495,6 +497,52 @@ UpdateMessageEvent updateMessageMoveEvent(
   );
 }
 
+/// [UpdateMessageEvent] where [origMessages] are moved to somewhere else.
+UpdateMessageEvent updateMessageEventMoveFrom({
```

Comment:
```
test [nfc]: Split eg.updateMessageMoveEvent helper.
```

Is there more we can do to focus this commit on the described change? I see that `eg.updateMessageMoveEvent` gets split into two functions, so I know that that function's callers will have to adapt. But I see other changes around the callers that don't seem necessary for that adaptation, and I'm finding it difficult to verify the refactor itself. For example, it looks like a `prepareOrigMessages` helper is introduced. That seems like it should be possible to do in a separate commit.

Reply:
Thanks for the advice. Extracted this part into `test [nfc]: Extract prepareOrigMessages`.